src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/hila/StreamBatch.java

```diff
@@ -0,0 +1,48 @@
+package de.zalando.aruha.nakadi.webservice.hila;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import de.zalando.aruha.nakadi.domain.Cursor;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Collections.unmodifiableList;
+
+@Immutable
+public class StreamBatch {
+
+    private Cursor cursor;
+    private List<Map> events = ImmutableList.of();
+
+    public StreamBatch(@JsonProperty("cursor") final Cursor cursor, @JsonProperty("events") final List<Map> events) {
```

Comment:
@jhorstmann good point, I will check it.
But I believe adding parameter-names module should be done as a separate task, not in a context of this one.